oemof_b3/tools/data_processing.py

```diff
@@ -643,6 +643,45 @@ def round_setting_int(df, decimals):
     return _df
 
 
+def postprocess_data(df_postprocessed, df_year, region, scenario, unit):
```

Comment:
I would exclude the concatenation step, to make the function serve a single purpose. This will make it easier to give a good name, like e.g. `prepare_b3_timeseries(df, **kwargs)`. You could use `format_header` to make sure that the columns obey the b3 convention.

Reply:
Done with commit https://github.com/rl-institut/oemof-B3/pull/199/commits/480d03e1d99159699c546f6799943ee5db107f23, https://github.com/rl-institut/oemof-B3/pull/199/commits/6dfbd89e5d0d31f23a9bcc0f667a9195a91fce25 and https://github.com/rl-institut/oemof-B3/pull/199/commits/766c48784f33cfa8731972fbdb482b333957efd2.